From b76559a406ff8a138f6f6c4be4f33ad4b7cea858 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Mon, 6 Mar 2017 17:32:29 +0800
Subject: [PATCH 1694/5242] MLK-14372: ASoC: fsl_rpmsg_i2s: remove unsupported
 sample rate

commit  5f1886ba6465e05c20acd0dea249f1c06f26170b from
https://source.codeaurora.org/external/imx/linux-imx.git

RTOS for M4 core in imx7ulp don't support 32kHz and 44kHz

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_rpmsg_i2s.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/sound/soc/fsl/fsl_rpmsg_i2s.c b/sound/soc/fsl/fsl_rpmsg_i2s.c
index 10c917e..5a66e90 100644
--- a/sound/soc/fsl/fsl_rpmsg_i2s.c
+++ b/sound/soc/fsl/fsl_rpmsg_i2s.c
@@ -27,7 +27,6 @@
 #include "imx-pcm.h"
 
 #define FSL_RPMSG_I2S_RATES	(SNDRV_PCM_RATE_8000|SNDRV_PCM_RATE_16000|\
-				SNDRV_PCM_RATE_32000|SNDRV_PCM_RATE_44100|\
 				SNDRV_PCM_RATE_48000)
 #define FSL_RPMSG_I2S_FORMATS	SNDRV_PCM_FMTBIT_S16_LE
 
-- 
1.7.9.5

